src/dev-app/legacy-button/legacy-button-demo.html

```diff
@@ -225,4 +139,9 @@ <h4 class="demo-section-header">Interaction/State</h4>
       </button>
     </div>
   </section>
+
+  <h4 class="demo-section-header">Overview</h4>
+  <section>
+    <button-overview-example></button-overview-example>
```

Comment:
This is using an example from the non-legacy button in the legacy button demo which is incorrect. We should either inline the example here or get rid of it completely.

Reply:
I ended up removing it since it's for the dev-app